Log level for invalid characters is too high

## Enhancement

```sql
mysql> create table t (a char(10) charset utf8mb4);
Query OK, 0 rows affected (0.03 sec)

mysql> insert into t values (0x8088);
ERROR 1366 (HY000): Incorrect string value '\x80' for column 'a'
```

```console
[ERROR] [column.go:321] ["incorrect string value"] [conn=403] [error="[table:1366]Incorrect string value '\\x80' for column 'a'"]
```

It is just a normal error and there is no need to log it in the server.

--- a/table/column.go
+++ b/table/column.go
@@ -318,7 +318,7 @@ func CastValue(ctx sessionctx.Context, val types.Datum, col *model.ColumnInfo, r
 		}
 	} else if err != nil && charset.ErrInvalidCharacterString.Equal(err) {
 		err = convertToIncorrectStringErr(err, col.Name.O)
-		logutil.BgLogger().Error("incorrect string value",
+		logutil.BgLogger().Debug("incorrect string value",
 			zap.Uint64("conn", ctx.GetSessionVars().ConnectionID), zap.Error(err))
 	}
 

